src/xdocs/report_issue.xml

```diff
@@ -99,16 +100,49 @@ private static final int SOMETHING = 1;
 Starting audit...
 Audit done.
 
+Expected: violation on first line.
+/var/tmp/Test.java:1:28: error: '{' is not preceded with whitespace.
         ]]></source>
       </p>
 
       <p>
-        Always specify what is expected.<br/>
+        For Windows users, please use "type" command instead of "cat".
+        Example of report that we expect:
 
-        <code>
-          Expected: violation on first line.
-          /var/tmp/Test.java:1:28: error: '{' is not preceded with whitespace.
-        </code>
+        <source><![CDATA[
+Check documentation: https://checkstyle.org/config_misc.html#Indentation
+
+D:\>type config.xml
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="Indentation"/>
+    </module>
+</module>
+
+D:\>type Test.java
+class Test {
+    void method() {
+        try { return; // false positive
+        }
+        catch (Exception e) { return; // false positive
```

Comment:
please convert it to false-nagative with message
`// no violation, but expected`

Reply:
I assume you mean just change the comment and expected output, right? For example purposes only.

Fixed.